gradle.properties

```
@@ -7,3 +7,5 @@ android.useAndroidX=true
 android.nonTransitiveRClass=true
 #MPP
 kotlin.mpp.enableCInteropCommonization=true
+# Buildkonfig
+buildkonfig.flavor=staging
```

As we will use the production API URL, so it's better to use `production` instead
```suggestion
buildkonfig.flavor=production
```

---

@suho I don't think so, because:
- I'm setting the default config as staging here https://github.com/luongvo/kmm-survey/pull/45/files#diff-cf9ae3b1eb7273a914f22b1e6d22d8447f88df0e7245a5b856fb751e507b979bR109-R126
- Staging env is using prod env, which does not mean the default config should be prod. We can set prod env to staging env instead. In the end, the idea is we will use exact staging config for staging env, and production config for prod env 🤔